docs/multicluster/upgrade.md

```diff
@@ -0,0 +1,49 @@
+# Antrea Multi-cluster Upgrade Guide
+
+The Antrea Multi-cluster feature is introduced from v1.5.0. There is no data-plane related changes in release v1.5.0, so Antrea deployment and Antrea Multi-cluster deployment are indenpendent. Please refer to the section `Antrea upgrade and supported version skew` in [Antrea Versioning](../versioning.md) to learn the requirement of Antrea upgrade. In this doc, it focuses on Multi-cluster deployment only.
+
+The goal is to support 'graceful' upgrades, it means it will have no significant disruption to data-plane connectivity between multiple member clusters, but there will be necessary disruption incurred by restart of individual components:
+
+* During Leader Controller restart, new member cluster, ClusterSet,ClusterClaim or ResourceExport will not be processed. Because the Controller also runs the validation webhooks for MemberClusterAnnounce, ClusterSet, ClusterClaim and ResourceExport.
+* During Member Controller restart, new ClusterSet or ClusterClaim will not be processed. Because the Controller runs the validation webhook for ClusterSet and ClusterClaim.
+
+The different Antrea Multi-cluster components need to support version skew. Since we introduce Multi-cluster feature from v1.5.0, our target or recommendation is to upgrade Antrea Multi-cluster to a new release up to 2 minor versions newer.
+
+* **Antrea Leader Controler**: must be upgraded first
+* **Antrea Member Controler**: must not be newer than the **Antrea Leader Controller**, and may be up to 2 minor versions older
```

Comment:
I feel a more natural way is to upgrade Leader Cluster first. Any issue from doing that?

Reply:
yes, here I mean to upgrade the Leader first, do you mean to change `Leader Controller` to `Leader cluster`?

Reply:
Ok. I must have misread the sentence. Sorry for that.